pipelines/coaddQAPlots.yaml

```diff
@@ -260,3 +260,18 @@ tasks:
         from lsst.analysis.drp.dataSelectors import (CoaddPlotFlagSelector, FlagSelector, GalaxyIdentifier,
         SnSelector, UnknownIdentifier)
         from lsst.analysis.drp.calcFunctors import MagDiff
+
+  plot_CircAp9_forced_skyObj_sky:
+    # Make a plot showing the 9-pixel radius circular aperture flux in units of nJy.
+    # This plot shows the on sky distribution of these values for sky objects.
+    class: lsst.analysis.drp.skyPlot.SkyPlotTask
+    config:
+      axisActions.zAction.column: "iAp9Flux"
+      connections.plotName: CircAp9_forced_skyObj_sky
+      selectorActions.flagSelector: FlagSelector
+      selectorActions.flagSelector.selectWhenTrue: ["merge_peak_sky"]
```

Comment:
All of the other plots have 
```
selectorActions.flagSelector: CoaddPlotFlagSelector
selectorActions.flagSelector.bands: ["I"]
```
Naively, I would assume you need the same here.

Reply:
These parameters are set as default, so not explicitly required here. However, I agree that explicitly naming them may help future-proof against any updates to the defaults. Will edit, test and push later today.

Reply:
Having tested these edits, I've reverted back. Adding the selector actions present in all other plots causes the pipetask run to crash here. This is because we require a `FlagSelector` action to select explicitly using the flag column `merge_peak_sky`. This logic is different than the other plots in this pipeline. The `FlagSelector` action does not have a `bands` parameter and, therefore, it cannot be specified here. In which case, I think the ticket as-stands is the correct way to go.